app/apps/client/admin/helpers.js

```diff
@@ -357,15 +357,19 @@ export const appStatusSpanProps = ({
 export const formatPrice = (price) => `\$${ Number.parseFloat(price).toFixed(2) }`;
 
 export const formatPricingPlan = (pricingPlan) => {
-	const perUser = pricingPlan.isPerSeat && pricingPlan.tiers && pricingPlan.tiers.length;
+	const { strategy, price } = pricingPlan;
```

Comment:
why not ?
```javascript
export const formatPricingPlan = ({ strategy, price, tiers }) =>
```
I didnt understand in some attributes we are using destructuring, in some attributes using `obj.attribute`